TreeDB/documentservice/rag_parity_test.go

```
@@ -0,0 +1,436 @@
+	message, _ := errObj["message"].(string)
+	return code, message
+}
+
+func ragParityCreateIndex(t *testing.T, handler *Handler, name string) {
+	t.Helper()
+	body := `{"name":"` + name + `","dimension":4,"scalar_fields":[{"field":"meta.tenant","value_type":"string"},{"field":"meta.priority","value_type":"int64"}]}`
+	status, payload := ragParityPost(t, handler, "/v1/indexes", body)
+	if status != http.StatusOK {
+		t.Fatalf("create index status=%d body=%s", status, mustJSON(payload))
+	}
+	info := payload["index"].(map[string]any)
```

golangci-lint (2.12.2)

[error] type assertion must be checked

(forcetypeassert)